### DISCUSSION TITLE
Downloaded ok but seeing _x_zm_retaid and x_zm_rhtaid on command line windows

### DISCUSSION THREAD

**Discussion by robmOz** [Battleman/zoomdl#67] [General]
Hi, just wondering if anyone is seeing this... ?

'_x_zm_rtaid' is not recognized as an internal or external command,
operable program or batch file.
'_x_zm_rhtaid' is not recognized as an internal or external command,
operable program or batch file.

after a successful download.

I am on Windows.

Thanks.

p.s. also noticed this message on this closed issue:

https://github.com/Battleman/zoomdl/issues/17

zoomdl.exe -u Downloading '<_**file location**_>\<_**file name removed**_>.mp4'
100%|█████████████████████████████████████████████████████████████| 309M/309M [01:45<00:00, 3.08MB/s]
Done!
'_x_zm_rtaid' is not recognized as an internal or external command,
operable program or batch file.
'_x_zm_rhtaid' is not recognized as an internal or external command,
operable program or batch file.
![_x_zm_  not recognised]
![_x_zm_  not recognised](https://user-images.githubusercontent.com/9070901/135782417-f3953994-bf1b-4487-8863-94860855cab4.png)

**Comment by alexlegresley** [comment]
The author mentioned in #17 that the url should be enclosed in double quotes.

**Comment by Battleman** [comment]
I'm not sure why I haven't seen this discussion until now. Yes indeed, what you see is a problem of quotes, that can't be addressed in zoomdl. Quotes are important, because some special characters are understood as additional commands. I'm no expert in Windows command line, but it appears just as in *NIX systems, the `&` character somehow indicates the start of a new command (in GNU/Linux, it asks to start the previous command in the background, roughly speaking, and execute the next command on the front). I guess it's something similar on Windows. So you are effectively starting the zoomdl command, and Windows passes to zoomdl the URL, until the first `&` character. Then it starts what follows as a new command (in that case, the command if `_x_zm_rtaid`, and so on with the next `&`. 

It appears from my understanding that on Windows systems you should use double quotes, but I'm still waiting on a more knowledgeable person to correct me on that.

**Reply by alexlegresley** [reply]
Escaping the reserved characters with `^` would work as well but both options seem to be parsed correctly in this instance, and double quotes seem more convenient. In my limited understanding of Windows, double quotes (and escape sequences in general in Windows) can have some rather nasty technicalities that start to matter more for applications like batch scripting. Like how in some instances double quotes will be preserved when escaping `echo "hello&world" -> "hello&world"` while the escape character will not have this problem `echo hello^&world -> hello&world`, which matters sometimes for variables.
